File: time_to_date.php
#!/usr/bin/php
<?php

echo "********使用说明********\n";
echo "1.输入参数 time_to_date 时间戳转换为日期格式 \n";
echo "2.输入参数 date_to_time 日期转换为时间戳格式 \n";
echo "3.输入参数 now_time 当前时间戳 \n";
echo "4.输入参数 exit 退出 \n";

do {
   fwrite(STDOUT,"请输入参数:");
   $agr = trim(fgets(STDIN));

   switch($agr)
   {
	case 'time_to_date':
	    fwrite(STDOUT,"请输入时间戳:");
            $time = trim(fgets(STDIN));
	   echo date('Y-m-d H:i:s',$time)."\n";
		break;
	case 'date_to_time':
            fwrite(STDOUT,"请输入日期:");
            $time = trim(fgets(STDIN));
           echo strtotime($time)."\n";
		break;
	case 'now_time':
           echo date('Y-m-d H:i:s')."\n";
		break;
   }
} while($agr != 'exit');

File: time_to_date.py
#!/usr/bin/python
# -*- coding: UTF-8 -*- 
import sys;
import time;

print "********使用说明********";
print "1.输入参数 time_to_date 时间戳转换为日期格式";
print "2.输入参数 date_to_time 日期转换为时间戳格式";
print "3.输入参数 now_time 当前时间戳";
print "4.输入参数 exit 退出";

is_while = 1;
while is_while:
  str = raw_input("请输入参数：");
  if str == 'time_to_date':
      time_stmp = input("请输入时间戳：");
      time_s = time.localtime(time_stmp);
      print time.strftime("%Y-%m-%d %H:%M:%S",time_s);
  elif str == 'date_to_time':
      tmp = input("请输入日期（1970-01-01 00：00：00）：");
      #time_s = time.localtime(tmp);
      print time.mktime(time.strptime(tmp,"%Y-%m-%d %H:%M:%S"));
  elif str == 'now_time':
      print time.time();
  elif str == 'exit':
	sys.exit(0);

